### PR TITLE
feat(butikfactory): DDS-928 Adding local push notifications

### PR DESCRIPTION
**Pull request by Pilot1258**
Adding local push notifications after uploading artifact

### CODE REVIEW

File: src/main/scala/ru/butik/butifactory/PushService.scala [hunk 1]
```diff
@@ -34,18 +34,18 @@ object PushService {
...
 }
 
-class PushService(apiKey: String) {
+class PushService(pushHost: String, pushPort: String, pushMethod: String, pushKey: String) {
   import com.twitter.conversions.time._
   private val twitter = Http.client
     .withRetryBackoff(Backoff.exponentialJittered(2.seconds, 60.seconds).take(10))
-    .withTransport.tls("fcm.googleapis.com")
-    .newService("fcm.googleapis.com:443")
+    .withTransport.tls(pushHost)
```
Comment (khamutov) [inline src/main/scala/ru/butik/butifactory/PushService.scala:41]: tls on 8081?

File: src/main/scala/ru/butik/butifactory/PushService.scala [hunk 2]
```diff
@@ -34,18 +34,18 @@ object PushService {
...
+    .newService(s"$pushHost:$pushPort")
 
   def pushDevice(deviceId: String, artifactVersion: ArtifactVersionAndroid): Future[Either[String, GoogleMessageResponse]] = {
-    val request = http.Request(http.Method.Post, "/fcm/send")
-    request.host = "fcm.googleapis.com"
+    val request = http.Request(http.Method.Post, pushMethod)
```
Comment (khamutov) [inline src/main/scala/ru/butik/butifactory/PushService.scala]: POST is a method, but pushMethod is a path

**Review by khamutov — CHANGES_REQUESTED** [review]

File: src/main/scala/ru/butik/butifactory/PushService.scala (hunk 2 above)
Comment (Pilot1258) [reply src/main/scala/ru/butik/butifactory/PushService.scala]: fixed

File: src/main/scala/ru/butik/butifactory/PushService.scala (hunk 1 above)
Comment (Pilot1258) [reply src/main/scala/ru/butik/butifactory/PushService.scala:41]: Yes

**Review by khamutov — APPROVED** [review]

**Pull request merged**